docs/working-with-flux.md

```diff
@@ -153,12 +153,12 @@ NAME   	NAMESPACE	REVISION	UPDATED                               	STATUS  	CHART
 podinfo	default  	1       	2022-05-27 01:44:35.24229175 +0000 UTC	deployed	podinfo-5.0.3	5.0.3
 ```
 
-Based on Karmada's propagation policy, you can schedule Helm releases to your desired cluster flexibly, just like Kubernetes scheduling Pods to the desired node.
+Based on Karmada's propagation policy, you can schedule Helm releases to your desired cluster flexibly, just like Kubernetes schedules Pods to the desired node.
 
 ## Customize the Helm release for specific clusters
 
 The example above shows how to distribute the same Helm release to multiple clusters in Karmada. Besides, you can use Karmada's OverridePolicy to customize applications for specific clusters.
```

Comment:
"distribute the same Helm release"
Does "distribute" mean the same thing as "propogate"? If yes, I would suggest using "propogate".